src/xrGame/ui/UIPdaWnd.cpp

```diff
@@ -125,6 +125,28 @@ void CUIPdaWnd::Init()
     UITabControl->SetAutoDelete(true);
     AttachChild(UITabControl);
     CUIXmlInit::InitTabControl(uiXml, "tab", 0, UITabControl);
+
+    const std::tuple<shared_str, shared_str> socIds[] = 
+    {
+        {"eptMap", "1"},
+        {"eptDiary", "2"},
+        {"eptContacts", "3"},
+        {"eptStatistics", "5"},
+        {"eptEncyclopedia", "6"}
+    };
+
+    for (u32 i = 0, len = UITabControl->GetTabsCount(); i < len; i++)
```

Comment:
Первый раз вижу чтобы так делали переменную len, буду знать!
Но это непривычно, неудобно читать. Да и код у нас тут выполняется один раз, так что можно без оптимизаций.
```suggestion
    for (u32 i = 0; i < UITabControl->GetTabsCount(); ++i)
```